### ISSUE TITLE
Code suggestions list doesn't receive keyboard focus when the list is short

### ISSUE THREAD

**Issue by ussuri** [GoogleChrome/chromedeveditor#3291]
The code suggestions list works well with the keyboard when there are enough items in it to trigger a scroller:

![screen shot 2014-08-28 at 4 15 40 pm](https://cloud.githubusercontent.com/assets/5606182/4084463/b5848f0e-2f09-11e4-9666-23fe522547ca.png)

Arrows move around the list and ENTER commits the selection. However, when the list is so short that there is no scroller, the focus stays in the editor, and the only way to select an item is using the mouse:

![screen shot 2014-08-28 at 4 16 18 pm](https://cloud.githubusercontent.com/assets/5606182/4084469/e69e8a90-2f09-11e4-8be2-8b7ad54142e4.png)
